cloudvolume/cloudvolume.py

```diff
@@ -640,8 +652,10 @@ def upload_image(self, img, offset):
     if str(self.dtype) != str(img.dtype):
       raise ValueError('The uploaded image data type must match the volume data type. volume: {}, image: {}'.format(self.dtype, img.dtype))
 
+    iterator = tqdm(self._generate_chunks(img, offset), disable=(not self.progress), desc='rechunking image')
```

Comment:
For consistence, you should match the order of the args (in `filteiter` you did `desc` then `disable`).
also capital R in `rechunking image`